hw6/tasks/task3.py

```diff
@@ -0,0 +1,33 @@
+"""
+Задача 3: Линейные уравнения +1
+
+Создайте функцию 'get_solution_linear_equation', которая возвращает решение
+линейного уравнения типа ax=b. Функция принимает 2 аргумента в следующем
+порядке:
+1. Вещественное число a - опциональный параметр, равный по умолчанию 5.0.
+2. Вещественное число b - опциональный параметр, равный по умолчанию 10.0.
+
+> При попытке деления на 0 нужно перехватить ошибку и вызвать исключение ошибки
+деления на 0 с текстом 'Деление на 0 невозможно'. В блоке except нужно явно
+указать тип ошибки деления на 0.
+
+Под блоком if __name__ == '__main__':
+1. Выполните функцию несколько раз: без аргументов, только с аргументом a,
+только с аргументом b, с обоими аргументами, с именованными аргументами.
+2. Распечатайте результат каждого вызова функции.
+"""
+
+
+def get_solution_linear_equation(a=5.0, b=10.0):
+    try:
+        return b / a
+    except ZeroDivisionError:
+        return "Деление на 0 невозможно"
```

Comment:
Здесь нужно вызвать такой же тип исключения: 
```suggestion
        raise ZeroDivisionError("Деление на 0 невозможно")
```